dockerfiles/init/modules/openshift/files/scripts/ocp.sh

```diff
@@ -153,15 +158,27 @@ run_ocp() {
 }
 
 deploy_che_to_ocp() {
-    #Repull init image only if IMAGE_PULL_POLICY is set to Always
-    if [ $IMAGE_PULL_POLICY == "Always" ]; then
-        docker pull "$IMAGE_INIT"
+    #Only generate scripts and config files if CHE_GENERATE_SCRIPTS=true
+    if [ $CHE_GENERATE_SCRIPTS == true ]; then
+      echo "OCP generating temporary scripts and configuration files at ${CONFIG_DIR}/instance/config/openshift/scripts/ ."
+      #Repull init image only if IMAGE_PULL_POLICY is set to Always
+      if [ $IMAGE_PULL_POLICY == "Always" ]; then
+         docker pull "$IMAGE_INIT"
+      fi
+      #wipeout config folder
+      docker run -v "${CONFIG_DIR}":/to_remove alpine sh -c "rm -rf /to_remove/" || true
+      docker run -t --rm -v /var/run/docker.sock:/var/run/docker.sock -v "${CONFIG_DIR}":/data -e IMAGE_INIT="$IMAGE_INIT" -e CHE_MULTIUSER="$CHE_MULTIUSER" eclipse/che-cli:${CHE_IMAGE_TAG} config --skip:pull --skip:nightly
+      cd "${CONFIG_DIR}/instance/config/openshift/scripts/"
+    else
+      echo "OCP using existing scripts in current folder."
```

Comment:
The previous message contains information about `configuration files` but this one doesn't.  Should we add it here too?
To be sure that I understand `configuration files` correctly: is it about yaml files?

Reply:
it is shell scripts and yaml files needed to deploy che multi user

Reply:
Added `configuration files` to second mesage.